drift-codec/src/test/java/io/airlift/drift/codec/metadata/TestThriftEnumMetadata.java

```diff
@@ -63,11 +70,15 @@ public void testDuplicateValues()
         thriftEnumMetadata(DuplicateValues.class);
     }
 
+    public enum MissingEnumAnnotation {}
```

Comment:
I think this should be an otherwise valid annotation so add an entry and the `@ThriftEnumValue` annotation